app/frontend/src/features/messages/requests/HostRequestSendField.tsx

```diff
@@ -35,6 +36,12 @@ export interface HostRequestSendFieldProps {
     Required<RespondHostRequestReq.AsObject>
   >;
 }
+enum ButtonTypes {
+  "button",
+  "submit",
+  "reset",
+  undefined,
+}
```

Comment:
`type ButtonTypes = "button" | "submit" | "reset" | undefined;` is more idiomatic. However just get rid of this (see other comments)